src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Automation/VisualBasic/VisualBasicVSImports.cs

```diff
@@ -68,8 +69,6 @@ public void Add(string bstrImport)
                         project.AddItem(ImportItemTypeName, bstrImport);
```

Comment:
I think here you want to wait for latest snapshot, see ReferencesHostBridge.ReceiveLatestSnapshotAsync as an example.

Reply:
As per the description, I'm going to leave that till later since this behaviour is the same as the current implementation, and I think its going to be better to move ReceiveLatestSnapshotAsync down to ProjectHostBridge, rather than copying it into here.

Reply:
Can we file a bug for this so that we don't lose track of it?

Reply:
Tried this on Friday and there wasn't a nice way to do make it public, so just going to do this in here after all :)